lib/liquider/ast.rb

```diff
@@ -69,16 +71,16 @@ def visit(visitor)
   CaseElseNode = Node.new_type(:case_else, :body)
 
   ForNode = Node.new_type(:for, :binding, :expression, :body) do
-    def reversed?
-      @reversed ||= options[:reversed] || Ast::BooleanNode.new(false)
+    def reversed
+      options[:reversed] || Ast::BooleanNode.new(false)
```

Comment:
Good question. I think supporting a superset of liquid is fine as long as it doesn't clash with something already valid in liquid. There is the issue of people relying on the fact that liquid just discards invalid stuff, but I think in this case it is fine (i.e. liquider can support expressions here).